docs/replatforming.md

```diff
@@ -37,6 +42,11 @@ _Examples:_
 De-risking guide content would have the front matter `tags: derisking`
 UX guide pages would have `tags: uxguide`
 
+## Base URLs
+Eleventy does not use `{{site.baseurl}}` to refer to other pages. When linking to another page on the site, use Eleventy's `url` filter as such:
+- For the home `index.md` page, use `{{ '[Markdown filename]' | url }}`.
+- For any other page in `content/[guide]`, use `{{ '../[Markdown file name]/' | url }}` (remember the trailing slash!)
```

Comment:
This is good for when the page you are linking to is in the same section as the current page, but not if it's in a different section (in which case you would need to use `../../`. I think'd be good to say something along the lines of "the path to the page relative to the current page". 

Same thought about possibly including an example here if it seems useful to you too.